src/qat/purr/qatconfig.py

```diff
@@ -0,0 +1,14 @@
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class QatConfig(BaseSettings):
+    """
+    Full settings for a single job. All values are defaulted on initialization.
+    """
+
+    model_config = SettingsConfigDict(env_prefix="QAT_")
+    MAX_REPEATS_LIMIT: int = 100_000
+    """Max number of repeats / shots to be performed in a single job."""
+
+
+qatconfig = QatConfig()
```

Comment:
I think we probably want to put

``` from qat.purr.qatconfig import qatconfig``` in qat/__init__.py so people can do `from qat import qatconfig` 

?

Reply:
Indeed, but that would be more of a general comment as there are some other functionalities which should be imported within `__init__` as well to allow for easier imports?